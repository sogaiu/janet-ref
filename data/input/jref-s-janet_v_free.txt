jref -s janet_v_free
