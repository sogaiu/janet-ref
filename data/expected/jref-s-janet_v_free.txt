#define janet_v_free(v)         (((v) != NULL) ? (janet_sfree(janet_v__raw(v)), 0) : 0)

/*
   janet_v_free
   +40 /home/user/src/janet/src/core/vector.h
*/

